packages/p2p-v2/types.ts

```diff
@@ -50,4 +46,62 @@ export type TCurrencyListItem = {
 
 export type TServerTime = ReturnType<typeof useServerTime>['data'];
 
-export type TOrders = NonNullable<ReturnType<typeof p2p.order.useGetList>['data']>;
+const prefix = '/cashier/p2p-v2';
+
+export type TRoutes = `${typeof prefix}/cashier/p2p-v2` | `${typeof prefix}`;
+
+// eslint-disable-next-line  @typescript-eslint/no-namespace
```

Comment:
can we add this to the top of the file? `/* eslint-disable @typescript-eslint/no-namespace */`? so you don't need to repeat it for each namespace

Reply:
oh yeaa